babel.config.json

```diff
@@ -49,18 +49,18 @@
           "~": "./src",
           "@@vap-svc": "./src/platform/user/profile/vap-svc",
           "@@profile": "./src/applications/personalization/profile",
-          "@department-of-veterans-affairs/component-library/LoadingIndicator": "@department-of-veterans-affairs/react-components/LoadingIndicator",
-          "@department-of-veterans-affairs/component-library/AlertBox": "@department-of-veterans-affairs/react-components/AlertBox",
-          "@department-of-veterans-affairs/component-library/ProgressButton": "@department-of-veterans-affairs/react-components/ProgressButton",
-          "@department-of-veterans-affairs/component-library/RadioButtons": "@department-of-veterans-affairs/react-components/RadioButtons",
-          "@department-of-veterans-affairs/component-library/Breadcrumbs": "@department-of-veterans-affairs/react-components/Breadcrumbs",
-          "@department-of-veterans-affairs/component-library/Modal": "@department-of-veterans-affairs/react-components/Modal",
-          "@department-of-veterans-affairs/component-library/FileInput": "@department-of-veterans-affairs/react-components/FileInput",
-          "@department-of-veterans-affairs/component-library/SystemDownView": "@department-of-veterans-affairs/react-components/SystemDownView",
-          "@department-of-veterans-affairs/component-library/Pagination": "@department-of-veterans-affairs/react-components/Pagination",
-          "@department-of-veterans-affairs/component-library/Checkbox": "@department-of-veterans-affairs/react-components/Checkbox",
-          "@department-of-veterans-affairs/component-library/DropDownPanel": "@department-of-veterans-affairs/react-components/DropDownPanel",
-          "@department-of-veterans-affairs/component-library/AdditionalInfo": "@department-of-veterans-affairs/react-components/AdditionalInfo"
+          "@department-of-veterans-affairs/component-library/LoadingIndicator": "@department-of-veterans-affairs/component-library/node_modules/@department-of-veterans-affairs/react-components/LoadingIndicator",
```

Comment:
These import aliases are only here to help pass tests that are using a `Shallow` render, there is about 73 tests that will fail if you remove these aliases. 

These tests need to be refactored to use `react-testing-library` and use `render()` instead

Reply:
I understand why the aliases are here, but why have they been changed to point to a dependency's `node_modules` directory?

Reply:
Here's some confirmation of my above comment:

1. I checked out an older commit (one from April 1)
2. Did a `yarn install`
3. `cd`'d into the `@department-of-veterans-affairs` `node_modules` namespace
4. Showed the packages there
5. We can see `react-components` and `web-components` in addition to `component-library`.

![Screenshot of the steps above in the terminal](https://user-images.githubusercontent.com/2008881/182458372-e549456b-142f-4097-823a-fcb7147d2780.png)

Reply:
When I was looking into this yesterday, I noticed that installing `@department-of-veterans-affairs/component-library` in other repos installs `react-components` and `web-components` at the top-level `department-of-veterans-affairs node_modules` just fine. It only seems to be a problem in `vets-website` for some reason. I ended up discovering that removing the `i18next-browser-languagedetector` dependency from `vets-website` installs `react-components` and `web-components` at the top-level, not sure why though. It looks like that dependency was added to the `component-library` [a few months ago](https://github.com/department-of-veterans-affairs/component-library/pull/346)

Reply:
Updating the aliases might be fine as a temporary fix to get this PR unblocked, my concern is if something unexpectedly changes the way the dependencies are being installed again. Then we'd be in the same position we're in now, having to update the aliases back